penney/functions.py

```diff
@@ -130,7 +132,8 @@ def C_calc(seq_dict):
     """
     C = []
     names = sorted(seq_dict)
-    p_seq = lambda seq: 1 / 2 ** len(seq)
+
+    def p_seq(seq): return 1 / 2 ** len(seq)
```

Comment:
Why this change it necassary? I believe using `lambda` is better here.

Reply:
This change was made by the `autopep8` script. It's not a bad change; using `lambda` functions in named format is uncommon.